doc/index.rst

```diff
@@ -29,7 +29,7 @@ Start the MTMount CSC as follows:
 
 Stop the CSC by sending it to the OFFLINE state.
 
-See MTMount `SAL communication interface <https://ts-xml.lsst.io/sal_interfaces/MTMount.html>`_ for commands, events and telemetry.
+See MTMount `SAL communication interface <https://ts-xml.lsst.io/sal_interfaces/mtmount.html>`_ for commands, events and telemetry.
 
 When MTMount is started (sent from STANDBY to DISABLED state) it launches a background process
```

Comment:
Maybe replace `When MTMount is started` with `When the MTMount CSC is started` here too?